app/views/styleguide/_dialogs.html.haml

```diff
@@ -1,2 +1,3 @@
 .sg-note
-  TODO: wizard, unsure of where else
+TODO: wizard, unsure of where else
```

Comment:
This line should not be changed at all. I recommend cleaning up the PR to include only one commit with the change to the other file.